src/QuadraticModels.jl

```diff
@@ -1,12 +1,6 @@
 module QuadraticModels
 
-using FastClosures
-using LinearOperators
-using NLPModels
-using Requires
-
-using LinearAlgebra
-using SparseArrays
+using FastClosures, LinearOperators, NLPModels, Requires, LinearAlgebra, SparseArrays
```

Comment:
I think it's good to separate standard modules from our own, and from those of others.